docs/_integration-with-exisiting-apps-swift.md

```diff
@@ -288,6 +302,28 @@ Wire up the new link in the main menu to the newly added event handler method.
 
 > One of the easier ways to do this is to open the view in the storyboard and right click on the new link. Select something such as the `Touch Up Inside` event, drag that to the storyboard and then select the created method from the list provided.
 
+##### 3. Window Reference
+
+Add an window reference to your AppDelegate.swift file. Ultimately, your AppDelegate should look something similar to this:
+
+```
```

Comment:
can you add the `swift` language tag here, so that the syntax results highlighted correctly?
Should be something like:
```
 ```swift
```